client/src/pages/forms/Confirmation.jsx

```diff
@@ -1,15 +1,20 @@
 // Global imports
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Confirmation = () => {
+const Confirmation = ({ reference }) => {
   return (
     <>
       <div className="govuk-panel govuk-panel--confirmation">
         <h1 className="govuk-panel__title">Account created</h1>
-        <div className="govuk-panel__body">We have sent you a confirmation email</div>
+        <div className="govuk-panel__body">COP reference:</div>
+        <div className="govuk-panel__body">
+          <strong>{reference}</strong>
+        </div>
       </div>
+      <p className="govuk-body">We have sent you a confirmation email</p>
```

Comment:
The screenshot includes a full stop on the end of this sentence:
```suggestion
      <p className="govuk-body">We have sent you a confirmation email.</p>
```